### DISCUSSION TITLE
Support for Chinese LPR

### DISCUSSION THREAD

**Discussion by shivangagarwal** [marcbelmont/deep-license-plate-recognition#111] [Q&A] [answered]
Hi,

Is there a plan to support Chinese LPR?

**Comment by danleyb2** [comment]
@shivangagarwal Yes we do support recognizing Chinese plates, do you have an image that is failing?

**Comment by marcbelmont** [comment, marked as answer]
Actually, the countries we support are listed here. We don't plan on adding China in the near future.
https://docs.platerecognizer.com/#country-codes

**Reply by shivangagarwal** [reply]
Thanks @marcbelmont
